<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HARDIConverterForm</class>
 <widget class="QWidget" name="HARDIConverterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="inputGroup">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="inputDataHLayout">
        <item>
         <widget class="QLabel" name="inputDataLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Data Set   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="inputDataCombo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="inputTypeHLayout">
        <item>
         <widget class="QLabel" name="inputTypeFixedLabel">
          <property name="text">
           <string>Data Set Kind  </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="inputTypeSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="inputTypeVarLabel">
          <property name="text">
           <string>&lt;None Selected&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="convGroup">
     <property name="title">
      <string>Conversion</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="outputKindLabel">
        <property name="text">
         <string>Output Data Set Kind</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="outputKindDSFRadio">
        <property name="text">
         <string>Discrete Sphere Function</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="convOptionsGLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="convTessLabel">
          <property name="text">
           <string>Tessellation Order</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="convTessSpin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="outputGroup">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="outputNewRadio">
        <property name="text">
         <string>New Data Set</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="outputOverwriteRadio">
        <property name="text">
         <string>Overwrite Existing Data Set</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="outputHLayout">
        <item>
         <widget class="QLabel" name="outputDataLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Data Set   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="outputDataCombo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="outputRenameHLayout">
        <item>
         <spacer name="outputRenameSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="outputRenameButton">
          <property name="text">
           <string>   Rename   </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="applyHLayout">
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>   Convert HARDI Data   </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="mainSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
